src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -600,7 +600,7 @@ void test2018MaineGovPrimaryDem() {
   @Test
   @DisplayName("testMinneapolisMultiSeatThreshold")
   void testMinneapolisMultiSeatThreshold() {
-    runTabulationTest("minneapolis_multi_seat_threshold");
```

Comment:
So this CVR has been altered to include precinct numbers. There are three files checked in: two precinct_summary.csv's and a precinct_summary.json. What exactly are we trying to get test coverage of by including those? Why only one .json?